sqlglot/optimizer/qualify_columns.py

```diff
@@ -599,6 +599,33 @@ def _expand_struct_stars(
     return new_selections
 
 
+def _expand_struct_stars_risingwave(expression: exp.Dot) -> t.List[exp.Alias]:
+    """[RisingWave] Expand/Flatten (foo.bar).*, where bar is a struct column"""
+    parens = t.cast(exp.Paren, expression.find(exp.Paren))
+
+    # check that last selected identifier is struct for nested structs
+    if not parens.this.is_type(exp.DataType.Type.STRUCT):
+        return []
+
+    struct_to_expand = exp.ColumnDef(this=parens.this, kind=parens.this.type)
```

Comment:
We don't need to construct a `ColumnDef` here, do we? Seems like we only need `kind`, i.e. `parens.this.type`, for line 614.